.circleci/download_google_fonts.py

```diff
@@ -82,7 +89,7 @@ def download(repo, family, types) :
 )
 
 download(
-    'https://github.com/expo/google-fonts/blob/master/font-packages/gravitas-one/',
+    'https://github.com/expo/google-fonts/blob/main/font-packages/gravitas-one/400Regular/',
```

Comment:
:100: :pray: Many thanks @emilykl 
Switching from `master` to `main` broke these. 
I suspect a similar problem would occur again.
I feel to avoid file changes (that occur on `master`/`main` branch of these repositories) it would be best to use permanent links for all these font files e.g.
`https://github.com/expo/google-fonts/tree/cc65b43125023a2ad696eeec6ec10763c35ff588/font-packages/gravitas-one/400Regular` instead of `https://github.com/expo/google-fonts/blob/main/font-packages/gravitas-one/400Regular/`.
Thank you!